docker/prometheus.yml

```diff
@@ -0,0 +1,14 @@
+global:
+  scrape_interval:     15s # By default, scrape targets every 15 seconds.
+
+  # Attach these labels to any time series or alerts when communicating with
+  # external systems (federation, remote storage, Alertmanager).
+  external_labels:
+    monitor: 'docs.rs-monitor'
+
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: 'docs.rs'
+    metrics_path: '/about/metrics'
+    static_configs:
+      - targets: ['web:3000']
```

Comment:
I don't know if `web` works on Docker for Windows, but `host.docker.internal` works on all platforms if you add
```yaml
extra_hosts:
  - "host.docker.internal:host-gateway"
```
to the compose file

Reply:
This is a docker-compose feature, not docker. Do you mind testing this on windows so we know if it works? I'd be very surprised if not.

Reply:
Yep, works on windows